.ci/updatecli.d/bump-upstream-agent-version.yml

```diff
@@ -0,0 +1,66 @@
+name: Upgrade the upstream agent version and related transitive dependencies
+# This task
+# - checks if the current otel-agent version referenced by the distro is the latest release
+# - if not, it will update the agent version AND the semconv and contrib versions to match the upstream
+
+scms:
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GITHUB_ACTOR" }}'
+      owner: elastic
+      repository: elastic-otel-java
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GITHUB_ACTOR" }}'
+      branch: main
+      commitusingapi: true
+
+sources:
+
+  upstream-agent-version:
+    kind: githubrelease
+    spec:
+      owner: open-telemetry
+      repository: opentelemetry-java-instrumentation
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      typefilter:
+        latest: true
+    transformers:
+      - trimprefix: "v"
+
+  distro-otel-agent-version:
+    kind: toml
+    scmid: githubConfig
+    spec:
+      file: gradle/libs.versions.toml
+      key: versions.opentelemetryJavaagentAlpha
+
+conditions:
+  update-required:
+    disablesourceinput: true
+    kind: shell
+    spec:
+      command: test "{{ source "upstream-agent-version" }}-alpha" != "{{ source "distro-otel-agent-version" }}"
+
+targets:
+
+  update-agent-version:
+    name: "Update upstream OpenTelemetry agent version and related dependencies"
+    conditionids: [update-required]
+    disablesourceinput: true
+    kind: shell # toml action currently erases comments, so we fallback to regex-replace
```

Comment:
required comment?

Reply:
Yes, but outdated. The [toml](https://www.updatecli.io/docs/plugins/resource/toml/) target would be perfect here, but we can't use it because it reformats the toml file and erases comments